### ISSUE TITLE
Integration with Doctrine ORM 2.5

### ISSUE THREAD

**Issue by Lctrs** [nicolopignatelli/valueobjects#35]
Hello,

Future 2.5 version of Doctrine ORM will provide support for ValueObjects through a new `@Embeddable` annotation.

I think it would be a good thing if your library provides the schema-mapping files (XML format would be the best). Do you agree with this ?

If yes, where do we put them in the directory ? I think a new directory `mappings/doctrine/orm` at the root of the projects could be a good idea.

I can work on a PR too once you give me your go. :)

**Comment by nicolopignatelli** [comment]
Ok for me, but keep in mind that only single VOs are supported embeddables, right now. For collections, you should still explicitly use one-to-many relationships.

I would use `resources/doctrine` as container directory for the mappings, since it's more compact and there will be no possibility to confuse those mappings with other resources.

**Comment by Lctrs** [comment]
OK.

Should we put `nullable="true"` for all columns ?

**Comment by nicolopignatelli** [comment]
No, I don't think so. We should put `nullable="true"` only for properties that actually accept null as value in PHP (but we have very few of them)

**Comment by nicolopignatelli** [comment]
Any update on the issue?

**Comment by liuggio** [comment]
If you start providing infrastructure choices would be awesome to add also the useful SF forms for the VOs
